firmware/tunerstudio/tunerstudio.template.ini

```diff
@@ -2309,7 +2309,7 @@ menuDialog = main
 		subMenu = hpfpPumpDialog,			"HPFP Pump Configuration", {hpfpCamLobes != 0}@@if_ts_show_gdi
 		subMenu = hpfpTargetTable,							"HPFP Target Pressure", {hpfpCamLobes != 0}@@if_ts_show_gdi
 		subMenu = hpfpCompensationTable,					"HPFP Pump Compensation", {hpfpCamLobes != 0}@@if_ts_show_gdi
-		subMenu = hpfpFuelMassCompensationTable,			"HPFP Fuel mass compensation factor", {hpfpCamLobes != 0 && injectionCompensationMode == 3}@@if_ts_show_gdi
+		subMenu = hpfpFuelMassCompensationTable,			"HPFP Fuel mass compensation factor", {hpfpCamLobes != 0 && injectorCompensationMode == 3}@@if_ts_show_gdi
```

Comment:
'3' is a magic constant, that's a code smell

can we reference that enum using @@templating@@ same as we reference other enum values?

Reply:
done, updated to `injectorCompensationMode != @@ICM_HPFP_Manual_Compensation@@`